projects/plugins/protect/changelog/update-protect-vulnerability-list-cta

```diff
@@ -0,0 +1,4 @@
+Significance: minor
+Type: changed
+
+Updated Protect vulnerability list CTA as per Figma designs
```

Comment:
Minor suggestion - I don't think we need to mention Figma designs in the changelog. We could do something a little more generic such as...
```
Updated the Protect upgrade CTA in threat list items.
```